src/main/java/com/znsio/teswiz/runner/LocalDevicesSetup.java

```diff
@@ -71,4 +76,49 @@ private static String getAdbCommandOutputFromLocalDevice(JadbDevice device, Stri
         LOGGER.info("\tOutput: " + adbCommandOutput);
         return adbCommandOutput;
     }
+
+    private static List<String> getBootedIOSSimulators() throws IOException {
```

Comment:
why are we throwing exceptions? Throw a custom runtime exception

Reply:
Removed it got added because of method chaining, have fixed the method from where it was getting generated.